### DISCUSSION TITLE
MSYS2 terminal no longer starts for me

### DISCUSSION THREAD

**Discussion by modusc896d352** [msys2/MSYS2-packages#4604] [General]
so, i opened up "MSYS2 MSYS" terminal
did a "pacman -Syuu"
terminal updated like 4 packages related to msys2 workingg (pacman and the like)
it prompted me to close the terminal and i obliged (pressed Y plus enter key after that)
i wanted to open the terminal again and it no longer opens up to me

i'm using windows 10 pro 22H2 btw

**Comment by lazka** [comment]
We had some cases where the update failed to stop all processes. A reboot should fix it for example.

**Reply by modusc896d352** [reply]
will consider rebooting

**Comment by modusc896d352** [comment]
rebooted, terminal finally opens now.

**Reply by Biswa96** [reply]
Actually, reboot is not required. You could just kill the msys2 process like mintty, bash, pacman, gpg etc. from task manager.
